webpack.config.js

```diff
@@ -29,7 +28,12 @@ module.exports = {
                     {
                       pattern: /%%PATH_TO_CONTROLLER%%/g,
                       replacement: function (match, p1, offset, string) {
-                        return controllerPaths[conf.pwt.adserver || "DFP"];
+                        if (conf.pwt.identityOnly == "1") {
+                            console.log("Returning controller path as -> IDHUB");
+                            return controllerPaths['IDHUB'];
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 9 tabs but found 28 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14881)